components/calendar/Header.tsx

```diff
@@ -154,7 +199,23 @@ export default class Header extends React.Component<HeaderProps, any> {
         <Button value="year">{locale.year}</Button>
       </Group>
     );
-
+    if (this.props.renderHeader) {
+      if (typeof this.props.renderHeader === 'function') {
+        return this.props.renderHeader({
```

Comment:
We don't check props type only if prop type changed and prev is deprecated.

Reply:
@zombieJ can you please add more details about this comment?

Reply:
Yes. Call directly.

```jsx
if (headerRender) {
  return headerRender(...);
}
```

Reply:
e.g.
If we have prop like 'test: string' and change it to `test: function` (and string be deprecated), it should check the type and add warning.
If it's new prop, no need to check.

Reply:
Got it thanks i will fix